core/lib/zksync_core/src/consistency_checker/tests/mod.rs

```diff
@@ -37,6 +37,7 @@ pub(crate) fn create_l1_batch_with_metadata(number: u32) -> L1BatchWithMetadata
 const PRE_BOOJUM_PROTOCOL_VERSION: ProtocolVersionId = ProtocolVersionId::Version10;
 const DIAMOND_PROXY_ADDR: Address = Address::repeat_byte(1);
 const VALIDATOR_TIMELOCK_ADDR: Address = Address::repeat_byte(23);
+const CHAIN_ID: u32 = 270;
```

Comment:
The chain id being 270, can we be sure about that?

Reply:
it is just a dummy value for unit test, the same as the constant `DIAMOND_PROXY_ADDR` above